src/store/modules/game.js

```diff
@@ -323,6 +324,26 @@ export default {
         );
       });
     },
+
+   async requestSpectate(context, gameId) {
+    return new Promise((resolve, reject) => {
+      io.socket.get(
+        '/game/spectate',
+        {
+          gameId,
+        },
+        function handleResponse(res, jwres) {
+          if (jwres.statusCode === 200) {
+            context.commit('updateGame', res);
+            context.commit('setMyPNum', 0);
+            return resolve();
+          }
+          return reject(new Error('Unable to spectate game'));
```

Comment:
You probably want the actual error message in here for debugging, unless this goes right to the snack bar.

Reply:
This does go right into the snackbar. The component `.catch()`'s the store's `dispatch()` and puts the error into a snackbar. Generally I like that paradigm as it keeps the errors human readable. I think troubleshooting the specific cause of the error is more of a development/debugging task that doesn't need to be user facing

Reply:
Makes sense, we can always log separately once you ever set me up that error logging account. Wink wink.

Reply:
https://github.com/cuttle-cards/cuttle/issues/368